### PR TITLE
Frontend Welcome Screen

### LINKED ISSUE #17: Create Welcome Screen
Need to create a welcome screen that has the abilities for users to:
- Login to their account
- Signup 
- Directly enter a survey id without the need to login

### PR DESCRIPTION
**Pull request by yehandesilva**
Closes #17 

Created Welcome Screen
image

Created Login Dialog
image

Created Registration Dialog
image

Currently, the user schema I've developed is:
- email (PK)
- firstName
- lastName
- password

In terms of integration, take a look at the /frontend/src/api directory. I've added comments in what I expect from the backend as well as the format of the input I will give. 
I will need 2 routes for this milestone:
- Create user
- Sign-in user

@AminZeina make the required routes for users as discussed in #19 and let @MichaelKyrollos know how they two should be properly integrated #20.

@pathumd Could you add the new logo images as requested in #22?

### CODE REVIEW

File: frontend/index.html
Comment (pathumd) [inline frontend/index.html:1]: Just a small thing here: On line 7, change the content of the <title> element to 'MonkeyPoll' (i.e. the space can be removed)

**Review by pathumd — COMMENTED** [review]
The code looks good to me! Just made really small formatting suggestion for your changes to index.html

**Review by pathumd — APPROVED** [review]
Changes look good now, nice work!

File: frontend/src/api/userApi.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        body: {
+            message: "Failed to login"
+        }
+    }
+
+    if (response.ok) {
+        status.success = true;
+        status.body = await response.json();
+    }
+
```
Comment (MichaelKyrollos) [inline frontend/src/api/userApi.js:28]: In the future, we might want to do a try catch block for this

File: frontend/src/api/userApi.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/*
+Login user given email and password.
```
Comment (MichaelKyrollos) [inline frontend/src/api/userApi.js:2]: related to https://github.com/yehandesilva/monkeyPoll/issues/20

**Comment by AminZeina** [comment]
Code itself looks good to me, UI is very clean! The endpoints in `/frontend/src/api` make sense to me; I'll implement the controller endpoints following this.

One comment I have is can we add documentation on how to run the frontend in our readme? Unless I missed it, cause I'm having trouble getting it up and running (plus this would be good to have either way). 

Also, reviewing our deliverables, our pom.xml should have everything needed to deploy, yet right now building our pom will only run the backend. Is there a way for us to integrate the frontend into our pom build? This would also "solve" my point above since running the frontend would simply be building and running as defined in the pom.

**Review by AminZeina — APPROVED** [review]
Thanks, with `npm run dev` I was able to run the frontend and looks great! Field validation seems to work well and everything we need for the welcome screen is present. 

My comment about updating documentation is covered in #27, and I've created #28 to track the issue of adding the frontend to the pom.xml build process.

**Pull request merged**